src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -145,6 +145,8 @@ internal static class IDEDiagnosticIds
         public const string UseNotPatternDiagnosticId = "IDE0083";
         public const string UseIsNotExpressionDiagnosticId = "IDE0084";
 
+        public const string RemoveRedundantEqualityWithTrueDiagnosticId = "IDE0085";
```

Comment:
```suggestion
        public const string RemoveRedundantEqualityWithTrueDiagnosticId = "IDE0090";
```

i'd prefer a new group here so we can have space to expand on the previous group if we want.